gpt_index/indices/postprocessor/node.py

```diff
@@ -82,3 +90,229 @@ def postprocess_nodes(
                 new_nodes.append(node)
 
         return new_nodes
+
+
+class BasePrevNextNodePostProcessor(BaseNodePostprocessor):
```

Comment:
nit: I prefer not to use inheritance for code re-use. I think we should keep the hierarchy as flat as possible unless you have a good reason.

Reply:
yeah, makes sense